resolwe_bio/processes/workflows/cut_and_run.yml

```diff
@@ -0,0 +1,624 @@
+- slug: workflow-cutnrun
+  name: Cut & Run
+  data_name: "{{reads|sample_name|default('?')}}"
+  requirements:
+    expression-engine: jinja
+  version: 1.0.0
+  type: data:workflow:cutnrun
+  category: Pipeline
+  description: |
+    Analysis of samples processed for high resolution mapping of DNA binding sites using
+    targeted nuclease strategy. The process is named CUT&RUN which stands for
+    Cleavage Under Target and Release Using Nuclease. Workflow includes steps of
+    trimming the reads with trimgalore, aligning them using bowtie2 to target species
+    genome as well as a spike-in genome. Aligned reads are processed to produce
+    bigwig files to be viewed in a genome browser. Peaks called using MACS2.
+    Fragmenting of reads is performed using alignmentSieve from
+    deeptools package.
+  input:
+    - name: reads
+      label: Input reads
+      type: data:reads:fastq:paired
+    - name: options_trimming
+      label: Trimming options (trimgalore)
+      group:
+        - name: quality_trim
+          label: Quality trimming
+          group:
+            - name: quality
+              label: Quality cutoff
+              type: basic:integer
+              required: false
+              description: |
+                Trim low-quality ends from reads based on Phred score.
+            - name: nextseq
+              label: NextSeq/NovaSeq trim cutoff
+              type: basic:integer
+              required: false
+              description: |
+                NextSeq/NovaSeq-specific quality trimming. Trims also dark cycles
+                appearing as high-quality G bases. This will set a specific quality
+                cutoff, but qualities of G bases are ignored. This can not be used
+                with Quality cutoff and will override it.
+            - name: phred
+              label: Phred score encoding
+              type: basic:string
+              required: false
```

Comment:
Is there a default value for this input in the tool? If yes, I think setting the default instead of `required: false` is a better approach. Also, the workflow inputs would have those values set explicitly. This applies also to other fields in this input block.

Reply:
Setting the values implicitly seemed like a good idea at the moment. Thinking about it more, setting explicit value makes it more reproducible, even though it may incur some extra  cost when modifying called processes.